### ISSUE TITLE
check if email is valid before sending invitation

### ISSUE THREAD

**Issue by giovanisp** [fairdemocracy/vilfredo-core#97]

**Comment by pietrosperoni** [comment]
This is linked to a discussion we had.
The problem is not if emails are valid, but if the email reply. Maybe saying the address does not exist.

If I send an email to cbdjscs@pippo.com this is an absolutely valid email in terms of grammar, but it is not associated to any person. In those case the email would bounce. We need to catch those bounce and say: hey, the email is not valid.

@giovanisp I shall assign this to you, so you check that this is correct. And you decide what milestone should we place this at

**Comment by giovanisp** [comment]
@derek, you can use this: https://pypi.python.org/pypi/validate_email to check if an email address really exists.
It should output things like as in these services: http://mailtester.com/testmail.php, https://mailboxlayer.com

**Comment by derekpa** [comment]
Yes I know, we did this in the previous version. Just not something we've had time to add yet.
